packages/models-library/src/models_library/progress_bar.py

```diff
@@ -12,8 +12,8 @@ class ProgressStructuredMessage(BaseModel):
     description: IDStr
     current: float
     total: int
-    unit: str | None
-    sub: "ProgressStructuredMessage | None"
+    unit: str | None = None
```

Comment:
I don't think that was the intention.
I think the tests are probably the problem. Please don't set things as optional when that is not the intention.

Reply:
this is ok